paper/paper.md

```diff
@@ -34,11 +34,17 @@ date: 22 July 2021
 bibliography: paper.bib
 ---
 
+# Statement of Need
+
+The most fruitful region of the radio spectrum to study the Early Universe is the microwave region. At those frequencies peaks the Cosmic Microwave Background (CMB) eletromagnetic radiation coming directly from the beginning of the Universe, just 370 thousand years into the 13.8 billion years age of the Universe.
+However, our own Milky Way galaxy also emits in the microwaves, for example through thermal radiation by interstellar dust grains or through synchrotron emission by relativistic electrons spiralling in magnetic fields.
```

Comment:
```suggestion
However, our own Milky Way galaxy also emits in the microwave, obscuring our view of the CMB.  Examples of this emission are thermal radiation by interstellar dust grains, and synchrotron emission by relativistic electrons spiralling in magnetic fields.
```

Reply:
should it be "in the microwaves" or in the microwave region?

Reply:
I would usually say "in the microwave", but perhaps that is a bit colloquial. "In the microwave portion of the spectrum" would be less colloquial. I would avoid "in the microwave region" as it's not clear we're referring to a region of a spectrum, and I don't think "in the microwaves" is correct.

Reply:
"In the microwave portion of the spectrum"